doc/client_lua_api.md

```diff
@@ -955,6 +955,12 @@ Can be obtained via `minetest.get_meta(pos)`.
     * `inventory`: `{list1 = {}, ...}}`
 
 -----------------
+### Definitions
+* `minetest.get_node_def(nodename)`
+    * Returns node definition table of `nodename`
+* `minetest.get_item_def(itemstring)`
+    * Returns item definition table of `itemstring`
```

Comment:
Do we perhaps need a documentation for these definition tables in this file too?